.travis.yml

```diff
@@ -35,6 +35,8 @@ env:
 
 matrix:
   include:
+    - php: 7.3
```

Comment:
not sure how to update the .travis.yml correctly because of:

```
#######################################################
# DO NOT EDIT THIS FILE!                              # 
#                                                     #
# It's auto-generated by symfony-cmf/dev-kit package. #
#######################################################
```

Reply:
@ElectricMaxxx has been doing the dev-kit thing. can you please help us?